### ISSUE TITLE
Compositing and Transparent Images

### ISSUE THREAD

**Issue by n4ru** [pebble/pebblejs#88]
Are either of these functioning? Setting 'compositing' to any value when creating an object causes it to not show up, and transparent images seem to have a black background instead of their transparency.

**Comment by Meiguro** [comment]
Yes, on basalt, compositing has not been fully implemented in the firmware, and 'set' has been overloaded to mean alpha blending. The other compositing modes are essentially meaningless.

Here is an example that is alpha blended on basalt:

```
var image = new UI.Image({
  position: new Vector2(0, 0),
  size: new Vector2(50, 50),
  image: 'images/some-image.png',
  compositing: 'set', // alpha-blended
});
```

You could find this information in the C SDK, but that is not obvious. Thanks for reminding me about this. I think I will alias some variant of 'alpha-blended' to be 'set' and set the default for basalt to be the new alias, and update the docs. The above code sample however currently works.
